app/views/layouts/districts/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1> <%= @district.name %></h1>
```

Comment:
This will create a space before the district name text. That might mess with your styling, just a heads up :)